dtfs-central-api/src/v1/routes/middleware/payload-validation/schemas/utilisation-report-csv-row-data.schema.ts

```diff
@@ -0,0 +1,9 @@
+import z from 'zod';
+
+const UtilisationReportCsvCellDataSchema = z.object({
+  value: z.string().nullable(),
+  column: z.string(),
+  row: z.union([z.string(), z.number()]),
+});
+
+export const UtilisationReportCsvRowDataSchema = z.record(z.string(), UtilisationReportCsvCellDataSchema);
```

Comment:
have some documentation here please.  Useful to see what z.record etc does on first glance if new

Reply:
I've added a docstring, cautious I don't want to describe what z.record() does however because it is a zod method and as such has it's own documentation: https://zod.dev/?id=records

Reply:
Would be great if we could add at least a little information about what it's checking exactly.

I agree we don't want to go into hugh depth about `z.record()` and `z.string()`, but equally, if you're not familiar with it, you have to go hunting around in external documentation. I think we should have a high level description in our own documentation, something simple like "checks that the object contains a string via zod methods" would be perfect

Reply:
Have added more detail